src/Microsoft.Management.UI.Internal/ShowCommand/ViewModel/CommandViewModel.cs

```diff
@@ -429,7 +429,7 @@ public string GetScript()
                 commandName = this.ModuleName + "\\" + commandName;
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\Get-ExperimentalFeature.Tests.ps1: line 177
177:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```